cognite_toolkit/cognite_modules/experimental/cdf_asset_source_model/default.config.yaml

```diff
@@ -1,6 +1,7 @@
 # Only valid for this module, loads template variables from environment
-model_space: ExtendedSourceDataModels
-instance_space: cdfTemplateInstances
+model_space: sp_ExtendedSourceDataModels
```

Comment:
I have commented on this before with mixed case approaches (which I don't think is a good idea), and the naming convention says that _ should be used as separator, so sp_extended_source_data_models.